docs/source/_json/workflow_get.json

```diff
@@ -0,0 +1,27 @@
+GET /plone/front-page/workflow
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "history": [
+    {
+      "action": null, 
+      "actor": "test_user_1_", 
+      "comments": "", 
+      "review_state": "private", 
```

Comment:
I guess both having in `GET /obj` as well as `GET /obj/workflow` is somewhat redundant. It's just that the current workflow state is one of the infos I would expect from the `workflow` endpoint as a user.